### ISSUE TITLE
Add support for PrometheusIntervalTreatment in Kubernetes Deployments

### ISSUE THREAD

**Issue by JulianLegler** [JulianLegler/oxn#14]
As a benchmark engineer, I want to use the PrometheusIntervalTreatment in my Kubernetes deployment to make the same assessments as in the docker deployment.

**Comment by JulianLegler** [comment]
Seems to be more complicated than expected in comparison to how it was done in docker. 

The Helm chart uses config maps. Changes to config maps are not automatically propagated to the pods. So Prometheus is not seeing the config changes directly, so the live reload functionality cannot be used this way. 

When restarting the pods, the new config maps are loaded correctly. Prometheus is currently not using persistent volumes, so all data is lost on pod restart. This means that we always lose our data, which can be limited to a one-time startup loss. When oxn collects the observable variables, the start time is no longer existing in the Prometheus pod, which produces errors. 

OXN has to apply the changes in another way. Ways to manipulate the config file directly in the pod may be not the best method but in this case a valid one.

**Comment by JulianLegler** [comment]
To resolve the aforementioned issues, I introduced support for persistent volume claims for Prometheus data. Now the pods can be restartet to apply the config map changes without data loss.

I also encountered that there is a category called "user config"->"scrapconfig" but it is empty. Also, there is this concept of ServiceMonitors, which is not employed by the astronomy shop. So it seems that the way of manipulating the ConfigMap and then restarting the pods seems to be okayish enough for the scope of this project.
